### PR TITLE
Added keyboard Punk75

### PR DESCRIPTION
**Pull request by dsanchezseco**
Added default keymap and my personal one for the punk75 keyboard(designed by me).

I'll publish the punk75 repo soon with the build guide and instructions

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/punk75/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            break;
+        case QMKURL:
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+    }
```
Comment (zvecr) [inline keyboards/punk75/keymaps/default/keymap.c:108]:
```suggestion
```

File: keyboards/punk75/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    _BASE,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QMKBEST = SAFE_RANGE,
+    QMKURL
+};
+
```
Comment (zvecr) [inline keyboards/punk75/keymaps/default/keymap.c:29]:
```suggestion
```

File: keyboards/punk75/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+void led_keypress_update(uint8_t led, uint16_t keycode, keyrecord_t *record) {
+    // When a key is pressed turn on the LED, when released turn it off
+    if (record->event.pressed) {
+      writePinHigh(led);
+      return;
+    }
+    else {
+      writePinLow(led);
+      return;
+    }
```
Comment (zvecr) [inline keyboards/punk75/keymaps/default/keymap.c:85]:
```suggestion
    writePin(led, record->event.pressed);
```

File: keyboards/punk75/punk75.h [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_grid( \
```
Comment (zvecr) [inline keyboards/punk75/punk75.h:29]:
```suggestion
#define LAYOUT_ortho_5x15( \
```

File: keyboards/punk75/readme.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+* Hardware Supported: ATmega32A-PU
+* Hardware Availability: None (yet) source it yourself ;)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make punk75:default
+
+To flash the keymap use
+
+    make punk75:default:USBasp
```
Comment (zvecr) [inline keyboards/punk75/readme.md:19]:
```suggestion
    make punk75:default:flash
```

File: keyboards/punk75/rules.mk [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (zvecr) [inline keyboards/punk75/rules.mk:37]:
```suggestion

LAYOUTS = ortho_5x15
```

File: keyboards/punk75/info.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+            "layout": [
+                {"label":"k00", "x":0, "y":0},
+                {"label":"k01", "x":1, "y":0},
+                {"label":"k02", "x":2, "y":0},
+                {"label":"k10", "x":0, "y":1, "w":1.5},
+                {"label":"k12", "x":1.5, "y":1, "w":1.5}
+            ]
+        }
+    }
+}
```
Comment (zvecr) [inline keyboards/punk75/info.json:18]:
```suggestion
{
    "keyboard_name": "punk75",
    "url": "",
    "maintainer": "dsanchezseco",
    "width": 15,
    "height": 5,
    "layouts": {
        "LAYOUT_ortho_5x15": {
            "layout": [
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},
                {"x":3, "y":0},
                {"x":4, "y":0},
                {"x":5, "y":0},
                {"x":6, "y":0},
                {"x":7, "y":0},
                {"x":8, "y":0},
                {"x":9, "y":0},
                {"x":10, "y":0},
                {"x":11, "y":0},
                {"x":12, "y":0},
                {"x":13, "y":0},
                {"x":14, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1},
                {"x":3, "y":1},
                {"x":4, "y":1},
                {"x":5, "y":1},
                {"x":6, "y":1},
                {"x":7, "y":1},
                {"x":8, "y":1},
                {"x":9, "y":1},
                {"x":10, "y":1},
                {"x":11, "y":1},
                {"x":12, "y":1},
                {"x":13, "y":1},
                {"x":14, "y":1},

                {"x":0, "y":2},
                {"x":1, "y":2},
                {"x":2, "y":2},
                {"x":3, "y":2},
                {"x":4, "y":2},
                {"x":5, "y":2},
                {"x":6, "y":2},
                {"x":7, "y":2},
                {"x":8, "y":2},
                {"x":9, "y":2},
                {"x":10, "y":2},
                {"x":11, "y":2},
                {"x":12, "y":2},
                {"x":13, "y":2},
                {"x":14, "y":2},

                {"x":0, "y":3},
                {"x":1, "y":3},
                {"x":2, "y":3},
                {"x":3, "y":3},
                {"x":4, "y":3},
                {"x":5, "y":3},
                {"x":6, "y":3},
                {"x":7, "y":3},
                {"x":8, "y":3},
                {"x":9, "y":3},
                {"x":10, "y":3},
                {"x":11, "y":3},
                {"x":12, "y":3},
                {"x":13, "y":3},
                {"x":14, "y":3},

                {"x":0, "y":4},
                {"x":1, "y":4},
                {"x":2, "y":4},
                {"x":3, "y":4},
                {"x":4, "y":4},
                {"x":5, "y":4},
                {"x":6, "y":4},
                {"x":7, "y":4},
                {"x":8, "y":4},
                {"x":9, "y":4},
                {"x":10, "y":4},
                {"x":11, "y":4},
                {"x":12, "y":4},
                {"x":13, "y":4},
                {"x":14, "y":4}
            ]
        }
    }
}
```

File: keyboards/punk75/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+ * | CAP LK | A      | S      | D      | F      | G      | HOME   | DEL    | PG UP  | H      | J      | K      | L      | ;      | ENTER  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+ * | LSHIFT | Z      | X      | C      | V      | B      | END    | UP     | PG DN  | N      | M      | ,      | .      | /      | RSHIFT |
+ * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+ * | LCTRL  | LGUI   | LALT   | FN     | SPACE  | SPACE  | LEFT   | DOWN   | RIGHT  | SPACE  | SPACE  | FN     | RALT   | RGUI   | RCTRL  |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+
+    [_BASE] = LAYOUT_grid( /* QWERTY */
```
Comment (zvecr) [inline keyboards/punk75/keymaps/default/keymap.c:45]:
```suggestion
    [_BASE] = LAYOUT_ortho_5x15( /* QWERTY */
```

File: keyboards/punk75/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | PREV   | PLAY   | NEXT   | STOP   | RGB SD | RGB SI | P4     | P5     | P6     | +      |        | RESET  |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | VOL-   | MUTE   | VOL+   | APP    | RGB VD | RGB VI | P1     | P2     | P3     | PENT   |        |        |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |        |        | RGB TG | FN     | RGB RMD| RGB MD | P0     |        | P.     | PENT   | PENT   | FN     |        |        |        |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+
+    [_FN] = LAYOUT_grid( /* FUNCTION */
```
Comment (zvecr) [inline keyboards/punk75/keymaps/default/keymap.c:67]:
```suggestion
    [_FN] = LAYOUT_ortho_5x15( /* FUNCTION */
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+    _DVORAK,
+    _QWERTY,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [_DVORAK] = LAYOUT_grid( /* DVORAK */
```
Comment (zvecr) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:31]:
```suggestion
    [_DVORAK] = LAYOUT_ortho_5x15( /* DVORAK */
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_GRV,   KC_BSLS, KC_EQL,  KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MUTE,
+        KC_TAB,  KC_QUOT, KC_COMM, KC_DOT,  KC_P,    KC_Y,    KC_LBRC,  KC_PSCR, KC_RBRC, KC_F,    KC_G,    KC_C,    KC_R,    KC_L,    KC_SLSH,
+        ESC_CTL, KC_A,    KC_O,    KC_E,    KC_U,    KC_I,    KC_HOME,  KC_MNXT, KC_PGUP, KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    KC_MINS,
+        KC_LSFT, KC_SCLN, KC_Q,    KC_J,    KC_K,    KC_X,    KC_END,   KC_MPLY, KC_PGDN, KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    KC_SFTENT,
+        KC_LCTL, KC_LGUI, KC_LALT, _______, MO(_FN), KC_SPC,  KC_SPC,   KC_MPRV, KC_BSPC, KC_BSPC, MO(_FN), _______, KC_RALT, KC_RGUI, KC_RCTL
+    ),
+
+    [_QWERTY] = LAYOUT_grid( /* QWERTY */
```
Comment (zvecr) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:39]:
```suggestion
    [_QWERTY] = LAYOUT_ortho_5x15( /* QWERTY */
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_GRV,   KC_BSLS, KC_EQL,  KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MUTE,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_LBRC,  KC_PSCR, KC_RBRC, KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_SLSH,
+        ESC_CTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_HOME,  KC_MNXT, KC_PGUP, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_END,   KC_MPLY, KC_PGDN, KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_MINS, KC_SFTENT,
+        KC_LCTL, KC_LGUI, KC_LALT, _______, MO(_FN), KC_SPC,  KC_SPC,   KC_MPRV, KC_BSPC, KC_BSPC, MO(_FN), _______, KC_RALT, KC_RGUI, KC_RCTL
+    ),
+
+    [_FN] = LAYOUT_grid( /* FUNCTION */
```
Comment (zvecr) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:47]:
```suggestion
    [_FN] = LAYOUT_ortho_5x15( /* FUNCTION */
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+void led_keypress_update(uint8_t led, uint16_t keycode, keyrecord_t *record) {
+    // When a key is pressed turn on the LED, when released turn it off
+    if (record->event.pressed) {
+      writePinHigh(led);
+      return;
+    }
+    else {
+      writePinLow(led);
+      return;
+    }
```
Comment (zvecr) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:66]:
```suggestion
    // When a key is pressed turn on the LED, when released turn it off
    writePin(led, record->event.pressed);
```

File: keyboards/punk75/keymaps/dsanchezseco/readme.md [hunk 14]
```diff
@@ -0,0 +1 @@
+# The default keymap for punk75
```
Comment (zvecr) [inline keyboards/punk75/keymaps/dsanchezseco/readme.md:1]:
```suggestion
# dsanchezseco keymap for punk75
```

File: keyboards/punk75/punk75.c [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+
+    return process_record_user(keycode, record);
+}
+
+bool led_update_kb(led_t led_state) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+    return led_update_user(led_state);
+}
+*/
```
Comment (zvecr) [inline keyboards/punk75/punk75.c:59]:
```suggestion
void matrix_init_kb(void) {
    // Set our LED pin as output
    setPinOutput(LED);

    matrix_init_user();
}

bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if(res) {
        writePin(LED, !led_state.caps_lock);
    }
    return res;
}
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+    [_FN] = LAYOUT_grid( /* FUNCTION */
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   DF(_QWERTY), _______, DF(_DVORAK), KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_RIGHT, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, _______, _______, _______, _______,
+        _______, _______, _______, KC_DOWN, KC_UP,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, MO(_FN), _______, _______, _______, KC_DEL,  KC_DEL,  MO(_FN), _______, _______, _______, _______
+    )
+
+};
+
```
Comment (zvecr) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:56]:
```suggestion

bool led_update_user(led_t led_state) {
    // disable keyboard level handling
    return false;
}

```

File: keyboards/punk75/config.h [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+#define MATRIX_COL_PINS { C2, C3, C6, C5, C4, A7, A6, A5, A4, B4, A3, B3, A2, B2, A1 }
+#define LED A0
+
+#define ENCODER1_PAD_A D0
+#define ENCODER1_PAD_B D1
+#define ENCODER2_PAD_A B1
+#define ENCODER2_PAD_B B0
+
+#define ENCODERS_PAD_A { ENCODER1_PAD_A, ENCODER2_PAD_A}
+#define ENCODERS_PAD_B { ENCODER1_PAD_B, ENCODER2_PAD_B }
```
Comment (zvecr) [inline keyboards/punk75/config.h:54]:
```suggestion
#define ENCODERS_PAD_A { D0, B1}
#define ENCODERS_PAD_B { D1, B0 }
```

File: keyboards/punk75/keymaps/default/keymap.c [hunk 18]
```diff
@@ -0,0 +1,95 @@
...
+    [_FN] = LAYOUT_ortho_5x15( /* FUNCTION */
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_NLCK, KC_SLSH, KC_ASTR, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+        KC_MSEL, KC_CALC, KC_MYCM, KC_MAIL, RGB_HUD, RGB_HUI, KC_P7,   KC_P8,   KC_P9,   KC_MINS, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS,
+        KC_MPRV, KC_MPLY, KC_MNXT, KC_MSTP, RGB_SAD, RGB_SAI, KC_P4,   KC_P5,   KC_P6,   KC_PLUS, _______, RESET,   _______, _______, _______,
+        KC_VOLD, KC_MUTE, KC_VOLU, KC_APP,  RGB_VAD, RGB_VAI, KC_P1,   KC_P2,   KC_P3,   KC_PENT, _______, _______, _______, _______, _______,
+        _______, _______, RGB_TOG, MO(_FN), RGB_RMOD,RGB_MOD, KC_P0,   _______, KC_PDOT, KC_PENT, KC_PENT, MO(_FN), _______, _______, _______
+    )
+};
+
+void led_keypress_update(uint8_t led, uint16_t keycode, keyrecord_t *record) {
```
Comment (fauxpark) [inline keyboards/punk75/keymaps/default/keymap.c:70]:
```suggestion
void led_keypress_update(pin_t led_pin, uint16_t keycode, keyrecord_t *record) {
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+bool led_update_user(led_t led_state) {
+    // disable keyboard level handling
+    return false;
+}
+
+void led_keypress_update(uint8_t led, uint16_t keycode, keyrecord_t *record) {
```
Comment (fauxpark) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:62]:
```suggestion
void led_keypress_update(pin_t led_pin, uint16_t keycode, keyrecord_t *record) {
```

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+    // When a key is pressed turn on the LED, when released turn it off
+    writePin(led, record->event.pressed);
```
Comment (fauxpark) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:64]:
```suggestion
    writePin(led_pin, record->event.pressed);
```

File: keyboards/punk75/readme.md [hunk 21]
```diff
@@ -0,0 +1,21 @@
+# punk75
+
+![punk75](https://github.com/dsanchezseco/punk75/raw/master/punk75_front.jpg)
```
Comment (fauxpark) [inline keyboards/punk75/readme.md:3]: Image 404s

File: keyboards/punk75/rules.mk [hunk 22]
```diff
@@ -0,0 +1,38 @@
...
+F_CPU =  16000000
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/punk75/rules.mk:14]:
```suggestion
# Processor frequency
F_CPU = 16000000

# Bootloader selection
```

File: keyboards/punk75/rules.mk [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```
Comment (fauxpark) [inline keyboards/punk75/rules.mk:36]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes        # Enable rotary encoders support
```

File: keyboards/punk75/readme.md [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+
+* Keyboard Maintainer: [dsanchezseco](https://github.com/dsanchezseco)
+* Hardware Supported: ATmega32A-PU
+* Hardware Availability: None (yet) source it yourself ;)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make punk75:default
+
+To flash the keymap use
```
Comment (fauxpark) [inline keyboards/punk75/readme.md:17]:
```suggestion
Flashing example for this keyboard:
```

File: keyboards/punk75/keymaps/default/keymap.c [hunk 25]
```diff
@@ -0,0 +1,95 @@
...
+    // When a key is pressed turn on the LED, when released turn it off
+    writePin(led, record->event.pressed);
```
Comment (fauxpark) [inline keyboards/punk75/keymaps/default/keymap.c:72]:
```suggestion
    writePin(led_pin, record->event.pressed);
```

File: keyboards/punk75/readme.md (hunk 21 above)
Comment (dsanchezseco) [reply keyboards/punk75/readme.md]: my bad, fixed

File: keyboards/punk75/keymaps/dsanchezseco/keymap.c [hunk 26]
```diff
@@ -0,0 +1,87 @@
...
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _DVORAK,
+    _QWERTY,
+    _FN
+};
+
+// Defines the keycodes used by our macros in process_record_user
+
```
Comment (fauxpark) [inline keyboards/punk75/keymaps/dsanchezseco/keymap.c:28]:
```suggestion
```

File: keyboards/punk75/config.h [hunk 27]
```diff
@@ -0,0 +1,73 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xDEED
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0001
+#define MANUFACTURER dsanchezseco
+#define PRODUCT punk75
+#define DESCRIPTION A custom 75 key ortho keyboard
```
Comment (fauxpark) [inline keyboards/punk75/config.h:28]:
```suggestion
```

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**